js/utils/templateHelpers.js

```diff
@@ -0,0 +1,24 @@
+var app = require('../App.js').getApp(),
```

Comment:
Decided to create this file since I had enough of passing certain things over and over to most templates (e.g. the server url). Loadtemplate will wire in the contents of the module so that I are provided as context to each template. For example, ob.moment && ob.cssImageUrl are now available in all templates.

If we find other things we are ad-nauseam passing into most templates, we could put it here. Also, we could put helper functions here like cssImageUrl, which given a hash and/or guid and/or fallback, will return a css style url that fetches from the get_image endpoint.